### PR TITLE
Feature/maptype staticmap

### PR DESCRIPTION
**Pull request by HelenGuov**
I am submit a PR for this issue #19 

Regards, 
Helen

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -27,12 +27,19 @@ class _MyAppState extends State<MyApp> {
...
   @override
   initState() {
     super.initState();
-    cameraPosition = new CameraPosition(Locations.portland, 12.0);
-    staticMapUri = staticMapProvider.getStaticUri(Locations.portland, 12,
-        width: 900, height: 400);
+    cameraPosition = new CameraPosition(Locations.portland, 2.0);
+    staticMapUri = staticMapProvider.getStaticUriWithMarkers(_markers,
+      maptype: StaticMapViewType.satellite, center: new Location(45.523970, -122.663081));
+    //staticMapUri = staticMapProvider.getStaticUri(Locations.portland, 12,
```
Comment (matthewtsmith) [inline example/lib/main.dart]: I'd prefer to leave this line uncommented and remove the line you added for testing the "center" attribute.

File: lib/static_map_provider.dart [hunk 2]
```diff
@@ -4,12 +4,14 @@ import 'package:map_view/location.dart';
...
 import 'map_view.dart';
 import 'locations.dart';
+import 'map_view_type.dart';
 
 class StaticMapProvider {
   final String googleMapsApiKey;
   static const int defaultZoomLevel = 4;
   static const int defaultWidth = 600;
   static const int defaultHeight = 400;
+  static const StaticMapViewType defaultMaptype =StaticMapViewType.roadmap;
```
Comment (matthewtsmith) [inline lib/static_map_provider.dart]: The formatting in this file looks off. Make sure to run dartfmt on it.

**Comment by HelenGuov** [comment]
@matthewtsmith  I have made the changes that you requested.

**Comment by HelenGuov** [comment]
Hey @matthewtsmith , is  any other changes that need to be done for this PR?

**Pull request merged**